core/client/src/main/java/alluxio/client/block/stream/UnderFileSystemFileOutStream.java

```diff
@@ -39,7 +39,7 @@
   public UnderFileSystemFileOutStream(FileSystemContext context, InetSocketAddress address,
       long ufsFileId) throws IOException {
     super(PacketOutStream
-        .createNettyPacketOutStream(context, address, -1, ufsFileId, Long.MAX_VALUE,
+        .createNettyPacketOutStream(context, address, -1, ufsFileId, Long.MAX_VALUE, -1,
```

Comment:
as far as the write tier goes, we could reserve 0 as the invalid value and change the "interpretOrdinal" logic accordingly